docker/Dockerfile.test

```diff
@@ -0,0 +1,57 @@
+FROM datajoint/djbase:py3.9-debian-fcd8909
+
+USER anaconda:anaconda
+
+COPY ./workflow-calcium-imaging/docker/apt_requirements.txt /tmp/
+RUN /entrypoint.sh echo "Installed dependencies."
+
+# Install Caiman
+RUN git clone --branch master https://github.com/datajoint-company/CaImAn 
+WORKDIR /main/CaImAn
+RUN conda install -n base -c conda-forge -y mamba
+RUN /bin/bash -c 'mamba env update --n base --file environment.yml' 
+# Suite2p requires np.__version__ == 1.21
+RUN pip install numpy==1.21
+RUN pip install .
+
+WORKDIR /main
+
+# Option 1 - Install DataJoint's remote fork of the workflow and elements
+# RUN git clone https://github.com/datajoint/workflow-calcium-imaging.git /main/
```

Comment:
```suggestion
# RUN git clone https://github.com/datajoint/workflow-calcium-imaging.git /main/workflow-calcium-imaging
```

Reply:
This is not necessary. Clone command will create the directory.